src/main/resources/templates/ftl-bits/app-js.ftl

```diff
@@ -1,13 +1,21 @@
 <!-- Include local javascript from js/... -->
-<script src="js/app.js"></script>
-<script src="js/controllers.js"></script>
-<script src="js/routes.js"></script>
-<script src="js/services.js"></script>
-<script src="js/mods.js"></script>
-<script src="js/constants.js"></script>
-
+<script type="text/javascript" src="js/app.js"></script>
+<script type="text/javascript" src="js/controllers.js"></script>
+<script type="text/javascript" src="js/routes.js"></script>
+<script type="text/javascript" src="js/services.js"></script>
+<script type="text/javascript" src="js/mods.js"></script>
+<script type="text/javascript" src="js/constants.js"></script>
 <!-- Controllers -->
-<script src="js/new-container-ctrl.js"></script>
-<script src="js/view-containers-ctrl.js"></script>
-<script src="js/upload-mod-ctrl.js"></script>
-<script src="js/manage-mods-ctrl.js"></script>
\ No newline at end of file
+<script type="text/javascript" src="js/new-container-ctrl.js"></script>
+<script type="text/javascript" src="js/view-containers-ctrl.js"></script>
+<script type="text/javascript" src="js/upload-mod-ctrl.js"></script>
+<script type="text/javascript" src="js/manage-mods-ctrl.js"></script>
+<!-- Tui2tone admin v.2 bootstrap -->
+<script type="text/javascript" src="js/navbar.js"></script>
+<script type="text/javascript" src="js/index.js"></script>
+<script type="text/javascript" src="js/alert.js"></script>
+<script type="text/javascript" src="js/button.js"></script>
+<script type="text/javascript" src="js/card.js"></script>
+<script type="text/javascript" src="js/theming.js"></script>
+<script type="text/javascript" src="js/chartjs.js"></script>
```

Comment:
Unfortunately, brute remove-and-try is what I would do...or the inverse, knowing what features of the theme are used, only add those in. That's what I do for all the optional core module of Angular.